packages/react-reconciler/src/ReactFiber.new.js

```diff
@@ -144,6 +144,8 @@ function FiberNode(
 
   // Effects
   this.effectTag = NoEffect;
+  this.subtreeTag = NoEffect;
+  this.deletions = [];
```

Comment:
Can we make this nullable instead of allocating an array?

Reply:
Done.